apps/server/src/routes/swagger/schemas.yml

```diff
@@ -66,62 +66,74 @@ components:
           description: Entities that are being processed
           items:
             type: string
-    Submission:
+
+    ActiveSubmissionResult:
       type: object
-      required:
-        - id
-        - categoryId
-        - entities
-        - state
-        - createdAt
-        - createdBy
       properties:
         id:
-          type: string
-          description: the id of the registration
-        categoryId:
-          type: string
-          description: the program short name this registration is for
-        entities:
-          type: object
-          description: submitted data
-          properties:
-            entityType:
-              $ref: '#/components/schemas/Entities'
+          type: number
+          description: Id of the Submission
         state:
           type: string
-          description: the state of the submission
+          description: State of the Submission
           enum: ['OPEN', 'VALID', 'INVALID', 'CLOSED', 'COMMITED']
-        createdBy:
+        organization:
           type: string
-          description: the user who created the submission
+          description: Organization the Submission belongs to
+        data:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
```

Comment:
why is this organized with `additionalProperties`, seems uneccesary?

Reply:
we use `additionalProperties` to refer to property names that are unknown, these properties or "keys" must be string an unique. It represents the `entity name`/`schema name` the records belong to.